src/ui/utils/environment.ts

```diff
@@ -29,3 +29,20 @@ export function isDeployPreview() {
 export function hasLoadingParam() {
   return url.searchParams.get("loading") != null;
 }
+
+export function getPausePointParams() {
+  const pointParam = url.searchParams.get("point");
+  const point = `${pointParam}`;
```

Comment:
You'll potentially end up with a param `point=null`, I'm pretty sure a URL with such a param (and hence trying to timeWarp to the ExecutionPoint `null`) would cause problems.
I think you should change the condition in line 43 to `pointParam && timeParam` instead.